### DISCUSSION TITLE
Authentication failures with Connection closed (proxy dest auth failed)

### DISCUSSION THREAD

**Discussion by mbyrdap** [Mailu/Mailu#3625] [User support]
So this is a fresh installation/domain. It's running the docker containers (all showing healthy), the DNS is setup and resolved correct, etc.
I can login to the admin webUI with the admin user and even an additional user.

Yet, when I try to login with IMAP, I get this showing up in the logs
```
imap-login: Info: Disconnected: Connection closed (proxy dest auth failed): user=<luser@domain.tld>, method=PLAIN, rip=30.124.14.31, rport=53172, lip=192.168.203.6, lport=993, TLS
```
I'm not sure where the ip 192.168.203.6 even comes from. The subnet defined in the environment file and docker-compose YAML is 172.17.0.1/16

Thanks in advance for any help

**Comment by dimonb** [comment]
this is because of configuration of allowed subnets, in my case it was subnet: 10.42.0.0/16 default, which break everything
